lib/WUI/nhttp/gcode_upload.cpp

```diff
@@ -17,6 +18,7 @@ extern "C" {
 // Inject for tests, which are compiled on systems without it in the header.
 size_t strlcpy(char *, const char *, size_t);
 size_t strlcat(char *, const char *, size_t);
+int mkdir(const char *path, mode_t mode);
```

Comment:
Prefer to use real headers for these? Or is there some problem, like mkdir not being available on some common systems?

Reply:
For some reason I was sure this is needed for tests to pass, but looking at it now, it isn't :sweat_smile:

Reply:
So, why is it still here? 😈